src/micro/attributes.html

```diff
@@ -12,51 +12,38 @@
 
 <script>
 
-  /**
-   * Support for `hostAttributes` property.
-   *
-   *     hostAttributes: 'block vertical layout'
-   *
-   * `hostAttributes` is a space-delimited string of boolean attribute names to
-   * set true on each instance.
-   *
-   * Support for mapping attributes to properties.
-   *
-   * Properties that are configured in `properties` with a type are mapped
-   * to attributes.
-   *
-   * A value set in an attribute is deserialized into the specified
-   * data-type and stored into the matching property.
-   *
-   * Example:
-   *
-   *     properties: {
-   *       // values set to index attribute are converted to Number and propagated
-   *       // to index property
-   *       index: Number,
-   *       // values set to label attribute are propagated to index property
-   *       label: String
-   *     }
-   *
-   * Types supported for deserialization:
-   *
-   * - Number
-   * - Boolean
-   * - String
-   * - Object (JSON)
-   * - Array (JSON)
-   * - Date
-   *
-   * This feature implements `attributeChanged` to support automatic
-   * propagation of attribute values at run-time. If you override
-   * `attributeChanged` be sure to call this base class method
-   * if you also want the standard behavior.
-   *
-   * @class base feature: attributes
-   */
-
   Polymer.Base.addFeature({
 
+    /**
+     * Static attributes applied to all instances of an element.
+     *
+     * If a custom element needs HTML attributes set on it during creation,
+     * these may be declared in a `hostAttributes` property on the prototype.
+     * This property is an object where keys match the attribute name, and
+     * values are converted into attribute values.
+     *
+     * If an attribute value is `true`, the attribute will be applied with no
+     * value, otherwise the value will be serialized to a string via
+     * `serialize()`.
+     *
+     * For example:
+     *
+     *     Polymer({
+     *       is: 'x-thing',
+     *       hostAttributes: {
+     *         tabindex: 0,
+     *         role: 'button',
+     *         'aria-disabled': true
```

Comment:
Note that in the case of ARIA, this should actually be a string-valued attribute, `'aria-disabled': 'true'`.